docs/src/main/paradox/scala/http/release-notes.md

```diff
@@ -5,7 +5,25 @@
 ### Support for HTTP(S) proxies with Authorization
```

Comment:
We should add another point that we now have documentation for the experimental Http2 support and link to it (and if it's not yet merge, let's finish it before the release).

Reply:
:+1: